### DISCUSSION TITLE
Only scanning microsoft and xbox series x,

### DISCUSSION THREAD

**Discussion by Predycate** [jef/streetmerchant#1000] [Ideas and feature requests]
Not scanning the other retailers in the uk, such as very, currys, or amazon uk 

<img width="960" alt="2020-11-28" src="https://user-images.githubusercontent.com/75165731/100514644-8146da80-316e-11eb-90fe-2b2264f53a46.png">

**Comment by Duck3y** [comment]
Currys appears to be bugged currently, i've never had any luck with Very. Amazon-uk appears to be working fine for other products, perhaps the product URL needs updating?

**Comment by poneill4497** [comment]
this is happening to me did you resolve this. only showing one retailer in uk for me

**Comment by Predycate** [comment]
@poneill4497, 
Not a clue on how to resolve this, it must be an on going issue when it comes to the uk sites, however the twitch streamer manages to get some of the uk sites to work so I guess, we won’t find out unless jef responds at some point

**Comment by poneill4497** [comment]
yea think the one of the URLs needs updated. I think thats why the sites are taking xbox webpages down to stop botting. they'll probably work once the sites make the xbox available to add to cart or something like that.
